### PR TITLE
Recordings: Frontend touch ups

### PR DESCRIPTION
**Pull request by scouillard**
## Description

Styled Recordings/Recordings list according to mock-ups.

## Screenshots (if appropriate):

image

### CODE REVIEW

File: app/assets/stylesheets/recordings.scss [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
+  }
+
+  .recording-icon-circle {
+    width: 40px;
+    height: 40px;
+    background-color: $primary-light;
+
+    svg {
+      padding-top: 12px;
+    }
```
Comment (farhatahmad) [inline app/assets/stylesheets/recordings.scss:14]: These can be done in Bootstrap

File: app/assets/stylesheets/recordings.scss [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
-    background: rgba(26, 69, 74, 0.2)
+.recordings-list {
+  th {
+    font-weight: normal;
+  }
+
+  .recording-icon-circle {
+    width: 40px;
+    height: 40px;
+    background-color: $primary-light;
```
Comment (farhatahmad) [inline app/assets/stylesheets/recordings.scss:10]: Can also be done in bootstrap

File: app/assets/stylesheets/recordings.scss [hunk 3]
```diff
@@ -1,11 +1,12 @@
...
-  font-weight: normal;
-}
 
-.recordings {
-  &:hover {
-    cursor: pointer;
-    background: rgba(26, 69, 74, 0.2)
+.recordings-list {
+  th {
+    font-weight: normal;
```
Comment (farhatahmad) [inline app/assets/stylesheets/recordings.scss:4]: Can be done in bootstrap

File: app/assets/stylesheets/recordings.scss (hunk 2 above)
Comment (scouillard) [reply app/assets/stylesheets/recordings.scss:10]: Doesn't seem to be working.
I could add a bg-primary-light class if you prefer.

**Review by farhatahmad — APPROVED** [review]

**Pull request merged**